infra/terraform/platform-base/main.tf

```diff
@@ -62,7 +62,7 @@ module "kops_spec" {
   cluster_dns_name = "${module.cluster_dns.dns_zone_domain}"
   cluster_dns_zone = "${module.cluster_dns.dns_zone_id}"
 
-  oidc_client_id  = "${var.oidc_client_id}"
+  oidc_client_id  = "${var.kubernetes_oidc_client_id}"
```

Comment:
I've checked the `tfvars` file and the Client ID looks correct 👍 

I'm not familiar with the other `oidc_client_id` value so not sure whether is used or not or how.

...oh I just saw [your comment](https://github.com/ministryofjustice/analytics-platform-ops/pull/333/files#r449076123) 👌